src/appmixer/openai/core/createCompletion/createCompletion.js

```diff
@@ -1,57 +1,26 @@
 'use strict';
 
-const lib = require('../../lib');
-
 module.exports = {
 
     receive: async function(context) {
+        const { content } = context.messages.in;
 
-        const { data } = await this.httpRequest(context);
-
-        return context.sendJson(data, 'out');
-    },
-
-    httpRequest: async function(context) {
+        context.log({ step: 'auth', auth: context.auth });
```

Comment:
remove this, we mustn't log auth info into logs

Reply:
of course, just a leftover from my development, thanks for catching that